includes/admin/views/html-admin-page-addons.php

```diff
@@ -113,21 +147,53 @@ class="<?php echo $current_section === $section->slug ? 'current' : ''; ?>"
 						?>
 						<li class="product">
 							<div class="product-details">
-								<?php if ( ! empty( $addon->image ) ) : ?>
-									<span class="product-img-wrap"><img src="<?php echo esc_url( $addon->image ); ?>" /></span>
+								<?php if ( ! empty( $addon->image ) || ! empty( $addon->icon ) ) : ?>
+									<span class="product-img-wrap">
+										<?php /* Show an icon if it exists, and an image otherwise */ ?>
+										<img src="<?php echo esc_url( empty( $addon->icon ) ? $addon->image : $addon->icon ); ?>" />
+									</span>
 								<?php endif; ?>
 								<a href="<?php echo esc_url( WC_Admin_Addons::add_in_app_purchase_url_params( $addon->link ) ); ?>">
 									<h2><?php echo esc_html( $addon->title ); ?></h2>
 								</a>
+								<?php if ( ! empty( $addon->vendor_name ) && ! empty( $addon->vendor_url ) ) : ?>
+									<div class="product-developed-by">
+										<?php
+											printf(
+												/* translators: %s vendor link */
+												esc_html__( 'Developed by %s', 'woocommerce' ),
+												sprintf(
+													'<a class="product-vendor-link" href="%1$s">%2$s</a>',
```

Comment:
It might be good to add `target="_blank"` for this one – unless you think it's not a good experience.

Reply:
That's a great suggestion! Thanks, @andfinally! Applied in https://github.com/woocommerce/woocommerce/pull/30840/commits/2461d20e2d51482a2ed83c3b97d9a44f8c0479cb.